cmd/rollouts-controller/main.go

```diff
@@ -267,6 +269,22 @@ func setLogLevel(logLevel string) {
 	log.SetLevel(level)
 }
 
+func createFormatter(logFormat string) log.Formatter {
+	var formatType log.Formatter
+	switch strings.ToLower(logFormat) {
+	case jsonFormat:
+		formatType = &log.JSONFormatter{}
+	case textFormat:
+		formatType = &log.TextFormatter{
+			FullTimestamp: true,
+		}
+	default:
+		formatType = &log.TextFormatter{}
```

Comment:
Logrus has its own default behavior to set the output to json or text or depending on a TTY. So changing the default explicitly to text will cause a change in behavior to everyone. I think it's best **not** to set a logformat in the default case and rely on logrus' default behavior.

Reply:
I have implemented the suggested change.